katharsis-jpa/src/main/java/io/katharsis/jpa/internal/query/AnyUtils.java

```diff
@@ -0,0 +1,118 @@
+package io.katharsis.jpa.internal.query;
+
+import io.katharsis.jpa.internal.meta.MetaAttribute;
+import io.katharsis.jpa.internal.meta.MetaDataObject;
+import io.katharsis.jpa.internal.meta.MetaLookup;
+import io.katharsis.jpa.query.AnyTypeObject;
+
+public class AnyUtils {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)